packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -22,28 +24,42 @@ type SelectInputProps = Pick<
   | 'autoFocus'
 > & {
   icon?: IconComponent;
+  onChange?: ({ name, values }: { name?: string; values: string[] }) => void;
 };
 
 const SelectInput = (props: SelectInputProps): JSX.Element => {
-  /**
-   * @TODO handle expanded state with Dropdown
-   */
-  const isPopupExpanded = false;
-  const value = '';
+  const {
+    isOpen,
+    value,
+    displayValue,
+    onSelectClick,
+    onSelectKeydown,
+    onSelectBlur,
+    dropdownBaseId,
+    activeIndex,
+    selectInputRef,
+  } = useDropdown();
```

Comment:
nope its in this PR only

Reply:
The storybook isn't working so wanted to confirm. The dropdown list items don't appear. I guess it's because you're adding action list in another PR?

![image](https://user-images.githubusercontent.com/6682655/211995215-ae66a44e-83f4-4a24-8584-83efed932a9b.png)

Reply:
Oh are you seeing the SelectInput's storybook? I added another Dropdown story with Dropdown.

SelectInput storybook is just for the SelectInput. I am not wrapping that in Dropdown right now.

I will update the Storybook docs at the end after things are built.

Reply:
I was confused because the story wasn't working 😸